www/qute.css

```diff
@@ -224,6 +224,34 @@ table td {
 	}
 }
 
+#toc {
+  margin-bottom: 2.5em;
```

Comment:
@phansch, why? This adds a very unnatural gap between the ToC and the following section:

[![screenshot-2017-06-17@08:29:27.png](https://s8.postimg.org/jevmkedc5/screenshot-2017-06-17_08_29_27.png)](https://postimg.org/image/6nhgdw3k1/)